src/HealthChecks.OpenIdConnectServer/DiscoveryEndpointResponse.cs

```diff
@@ -25,17 +25,21 @@ internal class DiscoveryEndpointResponse
     /// <summary>
     /// Validates Discovery response according to the <see href="https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata">OpenID specification</see>
     /// </summary>
-    public void ValidateResponse()
+    public void ValidateResponse(bool isDynamicOpenIdProvider = false)

```

Comment:
So far the validation was always performed. Disabling it could introduce a silent bug to some of our users. So to not break anyone, the default value should be `true`.